src/main/java/org/jenkinsci/test/acceptance/Matchers.java

```diff
@@ -201,7 +201,8 @@ public boolean matchesSafely(PageObject item) {
             @Override
             public boolean matchesSafely(final Jenkins jenkins) {
                 final User currentUser = jenkins.getCurrentUser();
-                return currentUser != null && currentUser.id().equals(user);
+                // if the user is not logged, currentUser can be not null with a null id
+                return currentUser != null && currentUser.id() != null && currentUser.id().equals(user);
```

Comment:
or just
```suggestion
                return currentUser != null && user.equals(currentUser.id());
```

I suppose

Reply:
that was my first intention, but after a search I couldn't be 100% sure that `user` will be always non-null. I can commit the suggestion if you prefer